vision/src/autogluon/vision/detector/detector.py

```diff
@@ -164,6 +168,17 @@ def fit(self,
                 Mini batch size
             lr : float
                 Trainer learning rate for optimization process.
+            early_stop_patience : int, default=-1
```

Comment:
Should we standardize that None is the way to disable? Since `-1` doesn't make sense for things like `early_stop_baseline` since metrics like r2 can have negative values.